tests/e2e/cluster_downgrade_test.go

```diff
@@ -130,11 +130,10 @@ func testDowngradeUpgrade(t *testing.T, clusterSize int) {
 }
 
 func newCluster(t *testing.T, clusterSize int) *e2e.EtcdProcessCluster {
-	epc, err := e2e.NewEtcdProcessCluster(context.TODO(), t, &e2e.EtcdProcessClusterConfig{
-		ClusterSize:  clusterSize,
-		InitialToken: "new",
```

Comment:
`InitialToken: "new"` is now redundant.
https://github.com/etcd-io/etcd/blob/2f558ca0dbf1217c1cc5b82a1e2aec428bf6a04f/tests/framework/e2e/cluster.go#L173-L178